README.md

```diff
@@ -15,12 +15,14 @@ QUICKINSTALL:
 
 ```
 # cd espressopp
-# cmake .
-# make
+# cmake -DEXTERNAL_BOOST=OFF -DEXTERNAL_MPI4PY=OFF .

```

Comment:
:-)))))))))))))

the good news is that they are installed.

the bad news is that they not always work, because of the heterogeneity of the machines, operating systems and other madnesses of the universe